### ISSUE TITLE
frio: fediverse address visible in header

### ISSUE THREAD

**Issue by hoergen** [friendica/friendica#5837]
I would love to see additionally the full fediverse address in the header of every post, because it is difficult to connect a mention, that is often only like `@nameblablablubb` and not the full name.

On the screenshot I added there should be `hoergen@horche.demkontinuum.de` to see what user from which instance is sending this.

![00_temp](https://user-images.githubusercontent.com/24474762/46568903-09dddd80-c94d-11e8-95e4-025fbf0b40d0.png)

I know you can see it, when you hover over the avatar, but it's anoying to do that with every account to find the right fediverse address (mention) to the name. It is not only about being practical, it is also about security, that you know there is no one spoofing with an avatar and name from another instance.

Like this example
![01_temp](https://user-images.githubusercontent.com/24474762/46568984-54ac2500-c94e-11e8-863a-f4f48f01c4d3.png)

### Friendica version you encountered the problem
git - develop

**Comment by hoergen** [comment]
As people are changing their names and avatars I "accidentally" deleted some contacts :(
Maybe we talk about that at the next Stammtisch. Think I have to do something with the templates.

**Comment by MrPetovan** [comment]
What about the hovercard on the contact avatar and name?

**Comment by hoergen** [comment]
I don't think we need to change the hover card.

**Comment by MrPetovan** [comment]
I mean, this piece of information already is available in the hovercard, I don't think we need to display it in every single post header.

**Comment by hoergen** [comment]
I know it is available on the hovercard, but we should display a reliable information directly withouth hovering or clicking somewhere.

**Comment by MrPetovan** [comment]
I don't agree, this is a rather technical information that isn't immediately necessary in each post. The mention autocomplete searches on display names as you type and will fill the Webfinger address for you.

**Comment by hoergen** [comment]
ok

**Comment by wakest** [comment]
I totally agree with @hoergen. I think it should be displayed with every post and its something I think all fediverse apps should have. I also disagree that there is anything technical about it. These are they handles we have to use to mention each other. Also the auto complete doesn't work quite often, and doesnt work for someone who is not logged in but sees a post and wants to interact with the person from another place with out having to copy or hover over anything, but by seeing the full handle.

**Comment by MrPetovan** [comment]
I'm actually turning around on this request, I've been needing to use the hovercard one to many times to obtain the webfinger address of a random contact when it probably should have been directly available in the first place.

**Comment by AlfredSK** [comment]
Please don't add too many lines there. We already have three lines (full name, post date and post location). Please don't add a fourth line with data we already have 1cm (0.39 inch) :-) away (in the hovercard).

**Comment by MrPetovan** [comment]
I thought about giving a single line for each piece of information, how do you like this:

> Hypolite Petovan
> via Friendica Developers
> hypolite@friendica.mrpetovan.com
> 11 minutes ago
> from DiCa
> (Brooklyn, USA)

**Comment by AlfredSK** [comment]
No, No, No. That's worse. Six lines in the header area of each post? Seriously? 
When I asked to not add a fourth line I didn't mean that it should be 6 lines. :-)

**Comment by AlfredSK** [comment]
As a comparison: Mastodon has 1 line. Hubzilla has two lines. Pleroma has 1 line. Diaspora has 1 line.

**Comment by annando** [comment]
I would like to keep it narrow.

**Comment by MrPetovan** [comment]
> No, No, No. That's worse. Six lines in the header area of each post? Seriously?

No. ☺

## Research

In the anonymous stream view,

Mastodon has 2 lines for 4 pieces of information:
- Display name
- Webfinger address
- Post date
- Interactions (reply, reshare, likes)

Diaspora has 1-3 lines for 2-4 pieces of information:
- Display name
- Post date
- Likes (optional)
- Reshares (optional)

Hubzilla has 2 lines for 2-3 pieces of information:
- Display name
- Post date
- Interactions (optional)

Twitter has 2 lines for 4 pieces of information:
- Display name
- Username
- Post date
- Interactions (numbers and buttons)

Friendica currently has 2-8 lines for 3-10 pieces of information:
- Display name
- Post date
- Contact network
- Origin app (Optional)
- Post location (Optional)
- Likes (Optional)
- Dislikes (Optional)
- Attend (Optional)
- Not attend (Optional)
- Maybe attend (Optional)

So the question isn't about adding a line, but more about removing extraneous information from the stream view (and leave them in the single post view) because they are too many at the moment.

Also I'm lacking the logged stream view details in Hubzilla, but I think this is enough to go on.

**Comment by hoergen** [comment]
I would love to keep it narrow too.
I created some examples . 
Example:
1. original (like it is now)
2. putting the fediverse address on the right side of the name
3. same like in 3, just smaller (doesn't look good)
4. fediverse address in the second line, then date/time, then send indicator
5. reducing to 2 lines putting 2 informations on each line.
![00_temp](https://user-images.githubusercontent.com/24474762/55039420-549b0100-5024-11e9-912e-9f4d3e325bb8.png)

**Comment by hoergen** [comment]
Or maybe on 5th example putting the send indicator on the first line after the name.

**Comment by MrPetovan** [comment]
My plan is to align the display with the global consensus for the anonymous stream display:
> Resharer Display Name  reshared
> ┌─┐**Display Name** _username@domain.tld_ Network [v]
> └─┘Post Date
<hr>

> [Post Content]
<hr>

> Interaction numbers (1 👍 0 👎 12 💬 1 ✔ 3 ❌ 4〰)

For the single top-level post display (Author info is displayed on the left column):

> Resharer Display Name reshared
> Post Date from App [v]
> Post Location
<hr>

> [Post Content]
<hr>

> Interaction numbers (1 👍 0 👎 12 💬 1 ✔ 3 ❌ 4〰)
> Interaction buttons (Like • Dislike • Comment • Attend • Not Attend • Maybe)

**Comment by annando** [comment]
BTW: Reshares will look different in the - hopefully - not so distant future. Then they are some information like the "likes", most likely summed up at the bottom.

**Comment by MrPetovan** [comment]
There's a difference between "This post has been reshared X times" and "This post is a reshare". The last one may actually not show in the single post display.

**Comment by annando** [comment]
A post could be shared by more than one of the users that your are following. The only way to display this, is to not use the "owner-id" anymore for displaying the resharer, but to completely display this as some information line.

When 3 people are resharing the same post, then the system doesn't display 3 different posts, but only adds the resharer information to it.

**Comment by MrPetovan** [comment]
So something along the lines of `Display Name and 3 others have reshared`?
